generated/nidaqmx/_library_interpreter.py

```diff
@@ -2941,12 +2941,12 @@ def get_physical_chan_attribute_bytes(self, physical_channel, attribute):
         with cfunc.arglock:
             cfunc.argtypes = [
                 ctypes_byte_str, ctypes_byte_str, ctypes.c_int32,
```

Comment:
FYI, https://github.com/ni/nidaqmx-python/pull/307 will fix the extra string parameter.